hedera-node/src/test/java/com/hedera/services/fees/TxnRateFeeMultiplierSourceTest.java

```diff
@@ -69,13 +70,32 @@ void setUp() {
 		subject = new TxnRateFeeMultiplierSource(mockProps, throttling);
 	}
 
+	@Test
+	void makesDefensiveCopyOfCongestionStarts() {
+		// setup:
+		final var someInstants = new Instant[] {
+				Instant.ofEpochSecond(1_234_567),
+				Instant.ofEpochSecond(2_234_567),
```

Comment:
If you push the next commit, please also fix this nit: 
Use 1_234_567L and 2_234_567L instead

Reply:
Done.